packages/sit-onyx/src/components/OnyxAppLayout/OnyxAppLayout.stories.ts

```diff
@@ -0,0 +1,125 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxAppLayout from "./OnyxAppLayout.vue";
+
+/**
+ * Layout component that structures your application.
+ * Includes space for a nav bar, page content, and overlays above the page or the whole app.
+ * Recommended to use on App.vue level of an application.
+ */
+const meta: Meta<typeof OnyxAppLayout> = {
+  title: "layout component/OnyxAppLayout",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxAppLayout,
+    events: [],
+    argTypes: {
+      navBar: {
+        control: { disabled: true },
+      },
+      default: {
+        control: { disabled: true },
+      },
+      pageOverlay: {
+        control: { disabled: true },
+      },
+      appOverlay: {
+        control: { disabled: true },
+      },
+    },
+    // storybook adds 1rem padding. The app layout fills the full available space
+    // so we need to counteract the padding with a negative margin.
+    decorators: [
+      (story) => ({
+        components: { story },
+        template: `
+          <div style="margin: -1rem;" >
```

Comment:
```suggestion
          <div style="margin: -1rem; font-family: var(--onyx-font-family); color: var(--onyx-color-text-icons-neutral-intense);">
```

Reply:
Jannick suggested to place font family settings on the layout component itself, now it's not needed additionally here :)

Reply:
So I guess the text / background color should also be set then?